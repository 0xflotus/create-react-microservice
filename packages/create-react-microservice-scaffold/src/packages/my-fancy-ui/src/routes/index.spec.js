// @flow

import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import IndexRoute from './index.js';

describe('<IndexRoute/>', () => {
  let props;

  beforeEach(() => {
    props = {
      name: 'foo'
    };
  });

  it('should render correctly.', () => {
    const wrapper = shallow(<IndexRoute {...props} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
