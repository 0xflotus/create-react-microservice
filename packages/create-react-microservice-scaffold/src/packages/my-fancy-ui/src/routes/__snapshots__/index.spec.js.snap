// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IndexRoute/> should render correctly. 1`] = `
<div
  className="mdl-layout__content"
  style={
    Object {
      "display": "flex",
      "justifyContent": "center",
    }
  }
>
  <div
    className="page-content"
    style={
      Object {
        "maxWidth": "640px",
      }
    }
  >
    <h2>
      Oii, foo!
    </h2>
    <Connect(CommentsList) />
  </div>
</div>
`;
